x-pack/plugins/serverless_search/public/layout/nav.tsx

```diff
@@ -42,10 +42,14 @@ const navigationTree: NavigationTreeDefinition = {
           title: i18n.translate('xpack.serverlessSearch.nav.devTools', {
             defaultMessage: 'Dev Tools',
           }),
-          isGroupTitle: true,
+          link: 'dev_tools:console',
+          getIsActive: ({ pathNameSerialized, prepend }) => {
+            return (
+              pathNameSerialized.startsWith(prepend('/app/dev_tools#/console')) ||
```

Comment:
you should be able to just do `pathNameSerialized.startsWith(prepend('/app/dev_tools'))`